tests/sync/test_settings.py

```diff
@@ -0,0 +1,36 @@
+import pathlib
+
+from kitovu.sync.settings import Settings, PluginSettings
+
+
+def test_load_a_sample_yaml_file():
+    settings = Settings.from_yaml_file(pathlib.PurePath('./tests/assets/smb_example_config.yml'))
+
+    expected_root_dir = pathlib.PurePath('~/Documents/HSR/semester_06')
```

Comment:
The `~` is a literal `~` here. You should call [expanduser](https://docs.python.org/3/library/pathlib.html#pathlib.Path.expanduser) or use [Path.home()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.home). Also, the real code should of course do the same.

Reply:
That's a good hint, I've added that